vibes/vibed-ui/src/render/legacy/SoftwareRenderer.cpp

```
@@ -0,0 +1,201 @@
+        case '+': { static const uint8_t g[7] = {0,0x04,0x04,0x1F,0x04,0x04,0}; return g; }
+        case '=': { static const uint8_t g[7] = {0,0x1F,0,0x1F,0,0,0}; return g; }
+        default:
+            return kGlyphUnknown;
+    }
+}
+
+} // namespace
+
+namespace render {
+
+SoftwareRenderer::SoftwareRenderer(int width, int height)
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: pixels

(cppcoreguidelines-pro-type-member-init)